pydocusign/client.py

```diff
@@ -47,6 +48,8 @@ def __init__(self,
             self.account_url = '{root}/accounts/{account}'.format(
                 root=self.root_url,
                 account=self.account_id)
+        #: Requests timeout
+        self.http_timeout = timeout
```

Comment:
Is there another timeout? Is there a reason for the `http_` prefix?
I think "timeout" is clear enough. We don't care about API calls being HTTP or whatever.
Argument name and attribute name should be consistent: if argument is "timeout", then attribute should be "timeout" as well.
So, imho: `s/http_timeout/timeout/`